components/automate-ui/src/app/entities/layout/layout.facade.ts

```diff
@@ -65,9 +64,6 @@ export class LayoutFacadeService implements OnInit, OnDestroy {
     this.sidebar$ = store.select(sidebar);
     this.showPageLoading$ = store.select(showPageLoading);
     this.updateDisplay();
-  }
-
-  ngOnInit(): void {
```

Comment:
This is the crux: getting rid of `ngOnInit` because this class is not a Component nor a Directive, so that `OnInit` hook is completely ignored. Moved it into the constructor instead so it gets executed.